orbit/pkg/osquery/flags.go

```diff
@@ -24,7 +24,7 @@ func FleetFlags(fleetURL *url.URL) []string {
 		"--distributed_tls_max_attempts=10",
 		"--distributed_tls_read_endpoint=" + path.Join(prefix, "/api/v1/osquery/distributed/read"),
 		"--distributed_tls_write_endpoint=" + path.Join(prefix, "/api/v1/osquery/distributed/write"),
-		"--logger_plugin=tls",
+		"--logger_plugin=tls,filesystem",
```

Comment:
Mhm... should we...

To be on the safe side, I'd leave `tls` only as `--logger_plugin` (folks can change this easily for the whole fleet from Fleet).

Reply:
But... now that I think of it, maybe this is a good idea, once osquery contacts fleet it should change to the default set by fleet config (tls).

Reply:
So in that case, should we also set the default there to be `tls,filesystem`? Otherwise whenever we ask for logs, we would have to ask them to set the plugin and wait.

Reply:
Just worried in case there's a reason we haven't set `filesystem` on the agent config at Fleet.

Let's open a separate issue to discuss that change.

Reply:
If we can differentiate between orbit and plain osquery, we can set filesystem in the former, but not on the latter... but I don't think we can differentiate at this time.

> Let's open a separate issue to discuss that change.

I think that if Fleet will override this the second it talks to orbit, then this issue is not really solved, because when an orbit user has issues, we will ask for logs and they won't have any. So it feels kind of related.

Reply:
> I think that if Fleet will override this the second it talks to orbit, then this issue is not really solved, because when an orbit user has issues, we will ask for logs and they won't have any. So it feels kind of related.

From: https://osquery.readthedocs.io/en/stable/deployment/logging/#status-logs
```
By default the osqueryd daemon sends INFO, WARNING, and ERROR logs to the configured logger plugins and to the process's stderr. You may configure this behavior using several flags documented in [CLI flags](https://osquery.readthedocs.io/en/stable/installation/cli-flags/).
```
Orbit logs osquery's stderr to a file.

> then this issue is not really solved

This issue was just about "orbit osquery" not crashing when users set `filesystem`.

Reply:
Ah... I misunderstood that :)